cluster-autoscaler/config/OWNERS

```diff
@@ -0,0 +1,4 @@
+reviewers:
+- ingvagabund
```

Comment:
I just ask some questions about updated README file. Don't think I can do anything else with the autoscaler in the time given.